docs/CODEOWNERS

```diff
@@ -2,5 +2,8 @@
 # request that modifies code that they own. Code owners are not automatically
 # requested to review draft pull requests.
 
+# Deafult
+* @ipfs/kubo-maintainers 
```

Comment:
> ```
> Unknown owner on line 6: make sure the team @ipfs/kubo-maintainers exists and has write access to the repository:
>     * @ipfs/kubo-maintainers
> ```

Reply:
that team didn't have write access, I just added it, should be goo dnow